style.qss

```diff
@@ -0,0 +1,17 @@
+/*
+Copyright (c) 2016 Shotgun Software Inc.
+ 
+CONFIDENTIAL AND PROPRIETARY
+ 
+This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit 
+Source Code License included in this distribution package. See LICENSE.
+By accessing, using, copying or modifying this work you indicate your 
+agreement to the Shotgun Pipeline Toolkit Source Code License. All rights 
+not expressly granted therein are reserved by Shotgun Software Inc.
+*/
+
+QLabel {
+    font-family: "Open Sans";
+    background-color: none;
+    border: none;
```

Comment:
I would love some comments for these style changes too, just explaining why we have to do this. I always find it useful with those kind of notes when you go back and revisit code at a later state and think "why did we do that again???"...